test/recipes/90-test_threads.t

```diff
@@ -20,13 +20,15 @@ use lib srctop_dir('Configurations');
 use lib bldtop_dir('.');
 
 my $no_fips = disabled('fips') || ($ENV{NO_FIPS} // 0);
+my $config_path = abs_path(srctop_file("test", $no_fips ? "default.cnf"
+                                                        : "default-and-fips.cnf"));
```

Comment:
Could this use fips-and-base.cnf ? (Or does it need something from default?)

Reply:
I just left what was there...

Reply:
There are references to "default" in the thread test program, although it only uses this in places where that isn't accessed.

I think it best to leave it as is to avoid confusion when the test is eventually updated in a way that breaks.

This test isn't about checking FIPS, it's about checking threading.